femitj/Quick-Credit
# #166107004 Post a loan repayment record endpoint

**What does this PR do?**

Admin can creates a loan repayment record in favor of a client

**Description of the task to be completed**

 - Ensure the following endpoints are working as they should

   - POST/api/v1/loans/<:loans-id>/repayment

**How should this be tested manually**

 - After you clone the repo,
 - cd into it and run npm install to install node modules.
 - Make sure you have nodejs and express installed.
 - Afterwards, run npm start to initiate the app.
 - test endpoint above using postman with localhost:8080 as the host
- authenticate with admin token
 - create with loan-id, amount, monthly installment, paid amount, balance 

**What are the relevant pivotal stories**
#166107004
